src/clusterfuzz/_internal/datastore/data_types.py

```diff
@@ -1114,6 +1114,18 @@ class TaskStatus(Model):
   time = ndb.DateTimeProperty()
 
 
+class WindowRateLimitTask(Model):
```

Comment:
Did you consider using redis for this? What would be the considerations that led to choosing datastore here? maintenance cost?

Reply:
I think there probably won't be much of a difference in practice between the two approaches. I think Redis might be a bit better for the short term (6 hour) rate limitng but probably worse for the 60-day rate limiting. WDYT?